x-pack/legacy/plugins/infra/public/components/metrics/sections/chart_section.tsx

```diff
@@ -112,7 +112,7 @@ export const ChartSection = ({
       <EuiTitle size="xs">
         <h3 id={section.id}>{section.label}</h3>
       </EuiTitle>
-      <div style={{ height: 250, marginBottom: 16 }}>
+      <div className="infMetricsExplorerChart" style={{ height: 250, marginBottom: 16 }}>
```

Comment:
I'm gonna change it to `infrastructureChart` since the class override isn't specific to Metrics Explorer anymore. That was just the class name that already exists.